src/lwm2m_coap_server_content.erl

```diff
@@ -22,23 +31,37 @@
 coap_discover(_Prefix, _Args) ->
     [].
 
-coap_get(_ChId, _Prefix, [], Query, _State) ->
+coap_get(_ChId, _Prefix, Query, _Content, State) ->
```

Comment:
was this a bug ? i.e. the changed order of arguments.

Reply:
In fact, the module never works correctly...This module has not worked properly since some long-ago PR changed the definition of the callback method in `lwm2m_coap_resource.erl`.  

BTW, I copied the order of the parameters and the return value type of these methods from emqx_lwm2m:emqx_lwm2m_coap_resource.erl module